hardware/gyros.py

```diff
@@ -29,6 +31,30 @@
 import json
 import re
 import threading
+import asyncio
+import random
+from hardware import u
+
+class MockGyro:

```

Comment:
Go to Northrop Grumman's website and look for an LN-200. This is the work-horse of the gyro world. Let's put it's geometry in here, and set the pitch to be a zero by default.